src/jobs/js_jest_runner_job.yml

```diff
@@ -2,7 +2,39 @@ description: Runs jest tests with CircleCI parallelism
 
 # Executor
 
-executor: js
+executor: << parameters.e >>
+
+# Parallelism
+
+parallelism: << parameters.parallelism >>
+
+# Parameters
+
+parameters:
+  e:
+    default: js_16
+    type: executor
+  # define which files should be included for testing
+  globPattern:
+    type: string
+  jest_options:
+    default: ""
+    type: string
+  # how many nodes will run in parallel
+  parallelism:
+    default: 4
+    type: integer
+  resource_class:
+    default: "medium+"
+    enum:
+      ["small", "medium", "medium+", "large", "xlarge", "2xlarge", "2xlarge+"]
+    type: enum
+  script:
+    type: string
```

Comment:
Personally I prefer to have parameters at the bottom of orb. For bigger orbs where params list might be big, I find it easier to scan the code if it starts directly with steps. Not a blocker though.